deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/dropout/IDropout.java

```
@@ -59,4 +62,13 @@
     void clear();
 
     IDropout clone();
+
+    /**
+     * Define the dropout for a {@link SameDiff} instance.
```

Maybe another "custom layers may throw unsupportedoperationexception if ..."